snafu/dns_perf_wrapper/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+RUN dnf install -y --nodocs python3 python3-pip epel-release && \
+    dnf install -y openssl-devel ck-devel yum-plugin-copr && \
+    dnf copr enable @dnsoarc/dnsperf -y && \
+    dnf install dnsperf -y && \
+    dnf clean all && rm -rf /var/cache/yum
+RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream && dnf clean all
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz -C /usr/bin/ oc && ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Why is python3 symlinked to python here?

Reply:
We symlink it to make sure python3 is used as the interpreter.